XLS-0066d-lending-protocol/README.md

```diff
@@ -0,0 +1,1560 @@
+| `LoanServiceFee`        |                    | `number`  |   `NUMBER`    |       0       | A nominal amount paid to the `LoanBroker.Owner` with every Loan payment.                                                                      |
+| `LatePaymentFee`        |                    | `number`  |   `NUMBER`    |       0       | A nominal funds amount paid to the `LoanBroker.Owner` when a payment is late.                                                                 |
+| `ClosePaymentFee`       |                    | `number`  |   `NUMBER`    |       0       | A nominal funds amount paid to the `LoanBroker.Owner` when an early full repayment is made.                                                   |
+| `InterestRate`          |                    | `number`  |   `UINT16`    |       0       | Annualized interest rate of the Loan in basis points.                                                                                         |
+| `LateInterestRate`      |                    | `number`  |   `UINT16`    |       0       | A premium added to the interest rate for late payments in basis points. Valid values are between 0 and 10000 inclusive. (0 - 100%)            |
+| `CloseInterestRate`     |                    | `number`  |   `UINT16`    |       0       | A Fee Rate charged for repaying the Loan early in 1/10th basis points. Valid values are between 0 and 100000 inclusive. (0 - 100%)            |
+| `PrincipalRequested`    | :heavy_check_mark: | `number`  |   `NUMBER`    |     `N/A`     | The principal amount requested by the Borrower.                                                                                               |
+| `StartDate`             | :heavy_check_mark: | `number`  |   `UINT32`    |     `N/A`     | The timestamp of when the Loan starts [Ripple Epoch](https://xrpl.org/docs/references/protocol/data-types/basic-data-types/#specifying-time). |
+| `PaymentTotal`          |                    | `number`  |   `UINT32`    |       1       | The total number of payments to be made against the Loan.                                                                                     |
+| `PaymentInterval`       |                    | `number`  |   `UINT32`    |      60       | Number of seconds between Loan payments.                                                                                                      |
+| `GracePeriod`           |                    | `number`  |   `UINT32`    |      60       | The number of seconds after the Loan's Payment Due Date can be Defaulted.                                                                     |
+
+##### 3.2.1.1 `Flags`
+
+| Flag Name           | Flag Value | Description                                     |
+| ------------------- | :--------: | :---------------------------------------------- |
+| `tfLoanOverpayment` |  `0x0001`  | Indicates that the vault supports overpayments. |
```

Comment:
Most transaction flags start with the high set of bytes, so this would preferably be:
```suggestion
| Flag Name           | Flag Value | Description                                     |
| ------------------- | :--------: | :---------------------------------------------- |
| `tfLoanOverpayment` |  `0x00010000`  | Indicates that the vault supports overpayments. |
```

Reply:
Done!